exercises/all-your-base/all_your_base_test.rb

```diff
@@ -0,0 +1,114 @@
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'all_your_base'
+
+class AllYourBaseTest < Minitest::Test
+  def test_single_bit_to_one_decimal
+    assert_equal [1], BaseConverter.new.convert(2, [1], 10)
+  end
+
+  def test_binary_to_single_decimal
+    skip
+    assert_equal [5], BaseConverter.new.convert(2, [1, 0, 1], 10)
```

Comment:
It would be nice to know what the numbers were.
I'd create 4 named local variables: `expected`, `input_base`, `digits`, `output_base`
So all your assertions become:
assert_equal expected, BaseConverter.convert( input_base, digits, output_base )